open_academy/tests/test_session.py

```diff
@@ -0,0 +1,68 @@
+from psycopg2.errors import NotNullViolation
+
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import ValidationError
+from odoo.tools import mute_logger
+
+
+class GlobalTestSession(TransactionCase):
+    """Session model test"""
```

Comment:
These comments are not needed, because the code explains them.

Reply:
Ok, I am going to change it!